controller/placeholder.js

```diff
@@ -129,6 +125,12 @@ function setup(placeholderService, should_execute) {
           results.filter(layersFilter).map(synthesizeDocs))
```

Comment:
You won't need `_.flatten` here anymore since `synthesizeDocs` now only ever returns a single results.